examples/kotlin/example/gradlew

```diff
@@ -0,0 +1,234 @@
+#!/bin/sh
```

Comment:
Instead of storing all of the build files in the repo, can't you run in a build script "gradle init" as a part of the example build process?

Reply:
It requires `gradle` to be intalled on the computer. With `gradlew` you can bypass that and get any version of gradle at compile time.